test/integration-test/PassStageMerger/Test01_after_ref.json

```diff
@@ -1,145 +1,133 @@
 {
   "IIR": {
     "Stencil0": {
-      "Fields": [
-        {
+      "Fields": {
+        "field_a0": {
           "IsTemporary": false,
           "dim": "[1,1,1]",
           "field": {
-            "accessID": 17,
+            "accessID": 15,
             "extents": {
               "read_access": "[(0, 0), (0, 0), (0, 0)]",
               "write_access": "null"
             },
             "intend": 2,
             "interval": "{ Start : End }",
-            "name": "field_b0",
             "redundant extents": {
               "read_access": "[(0, 0), (0, 0), (0, 0)]",
               "write_access": "null"
             }
-          },
-          "name": "field_b0"
+          }
         },
-        {
+        "field_a1": {
           "IsTemporary": false,
           "dim": "[1,1,1]",
           "field": {
-            "accessID": 18,
+            "accessID": 16,
```

Comment:
it seems that from testing some ID's are not generated correctly

Reply:
what do you mean, this is just the reordering, that we now fixed (and is different than the ref in master)